pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/ConnectionTimeouts.java

```diff
@@ -18,36 +18,25 @@
  */
 package org.apache.pinot.client;
 
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+
 /**
  * Connections time out for AsyncHttpClient
  */
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
```

Comment:
(minor) not introduced in this PR, but let's reformat this file to fix the indentation